gdpr/docs/how-to-integrate-to-Helsinki-profile-with-Tunnistamo.md

```diff
@@ -0,0 +1,62 @@
+
+## 2. Setup the clients and their scopes in Tunnistamo
+
+### Communication between the Culture kids apps and the Tunnistamo
+
+> The Culture Kids apps are already communicating with the Tunnistamo: The user authenticates from the public UI or from the admin UI and then the user's profile is read from the Kukkuu API. The JWT token is read by the django-helusers which is installed in the Kukkuu API and it will take care of updating the user information to the Kukkuu API database.
+
+The most important Tunnistamo configurations for the Culture kids services are described in [Culture Kids with Tunnistamo](tunnistamo-kukkuu-configuration.md).
+
+### Communication between the Helsinki-profile and the Tunnistamo
+
+> The Helsinki-profile is the service being integrated to the Culture Kids service, so at this point there are no configurations between the Helsinki-profile API or Helsinki-Profile UI and the Tunnistamo. Also the UI needs to communicate properly with the API.
+
+The most important Tunnistamo configuration for The Helsinki-Profile services are described in [Helsinki-Profile with Tunnistamo](tunnistamo-profile-configuration.md).
+
+### Communication from Helsinki-profile to the Culture Kids through Tunnistamo
+
+> When the Culture Kids and the Helsinki-Profile are already communicating with Tunnistamo (the user can successfully login from the UI clients), the only part missing in order to take advantage of the GDPR API, is to add the GDPR API scope configurations to the Tunnistamo.
+
+> NOTE: To configure the actual UI clients and the API servers to use the Helsinki-profile, see the [Setup the Helsinki-Profile](#3-setup-the-helsinki-profile) and the README files of the [application repositories](#1-download-and-install-all-the-needed-services).
+
+To configure the Helsinki-Profile GDPR API client scopes in Tunnistamo, see the [Culture Kids with Helsinki-Profile GDPR API through Tunnistamo](tunnistamo-kukkuu-configuration.md).
+
+## 3. Setup the Helsinki-Profile
+
+In order to get the Helsinki-Profile UI to work with API, The Helsinki-Profile services needs to be set so that they match with the Tunnistamo client ids. Also, the Helsinki-Profile API itself needs an internal service.
+
+To setup the Helsinki-Profile for internal use and for the Helsinki-Profile UI, see the [Setup the Helsinki-Profile for internal use and for UI](setup-helsinki-profile.md#configure-the-internal-service-for-helsinki-profile-ui-and-internal-graphql-ui).
+
+To setup the Helsinki-Profile for Culture Kids use, see the [Setup the Helsinki-Profile for internal use and for UI](setup-helsinki-profile.md#configure-the-culture-kids-service).
+
+## 4. Test that all the communication works
+
+Dependencies for the user test cases:
+
+1. The profile needs to have some connected services. An user profile can add more services to the list of the connected service from the Helsinki-Profile UI.
```

Comment:
typo: An user -> A user (because "An" article is used with words starting with a vowel sound when pronounced, "A" is with words starting with consonant sounds and "user" is pronounced [ˈyü-zər](https://www.merriam-webster.com/dictionary/user?pronunciation&lang=en_us&dir=u&file=user0001) i.e. it starts with a consonant sound)